src/app.js

```diff
@@ -495,10 +495,10 @@ app.all("/manifest.webmanifest", async (req, res) => {
   }));
   res.json(Object.assign({ shortcuts }, manifest));
 });
-
-app.all("/:player/:profile?", async (req, res, next) => {
-  res.redirect(`/stats${req.path}`);
-});
+//
```

Comment:
same with this

Reply:
this one may lead to some unexpected redirect, so comment it too